packages/rest/src/parser.ts

```diff
@@ -78,29 +85,47 @@ export async function parseOperationArgs(
 async function loadRequestBodyIfNeeded(
   operationSpec: OperationObject,
   request: Request,
-): Promise<MaybeBody> {
-  if (!operationSpec.requestBody) return Promise.resolve();
+): Promise<RequestBody> {
+  if (!operationSpec.requestBody) return Promise.resolve({value: undefined});
 
   const contentType = getContentType(request);
   debug('Loading request body with content type %j', contentType);
+
+  if (
+    contentType &&
+    contentType.startsWith('application/x-www-form-urlencoded')
+  ) {
+    const body = await parseFormBody(request).catch((err: HttpError) => {
+      debug('Cannot parse request body %j', err);
+      err.statusCode = 400;
+      throw err;
+    });
+    // form parser returns an object with prototype
+    return {
+      value: Object.assign({}, body),
+      coercionRequired: true,
+    };
+  }
+
   if (contentType && !/json/.test(contentType)) {
     throw new HttpErrors.UnsupportedMediaType(
```

Comment:
This branch is not covered by any test, see https://coveralls.io/builds/19498795/source?filename=packages%2Frest%2Fsrc%2Fparser.ts

Please add a new test, for example sending a `text/plain` payload (request body).